posts/reflections/Lessons_from_building_Krets.md

```diff
@@ -0,0 +1,70 @@
+# Lessons from building Krets
+## Trying to create a profitable project alone
+
+I've spent most of my time between summer and christmas 2020 working on [Krets](https://krets.app). The project was 
```

Comment:
I* ?

Reply:
Utdype? :) Jeg skjønner ikke

Reply:
Lurte på om det er I spent most of my time. istedenfor I have, men det er et spørsmålstegn der så er ikke så sikker :P

Reply:
Usikker jeg også :)